src/django/api/serializers/log_download_query_params.py

```
@@ -6,6 +6,8 @@
 )
 from api.models.download_log import DownloadLog
 
+from api.models.download_log import DownloadLog
```

Ruff (0.11.9)

Redefinition of unused `DownloadLog` from line 7

Remove definition: `DownloadLog`

(F811)